CryptoBinPkg/CryptoBinPkg.dsc

```diff
@@ -45,6 +45,7 @@
   DebugPrintErrorLevelLib|MdePkg/Library/BaseDebugPrintErrorLevelLib/BaseDebugPrintErrorLevelLib.inf
   DevicePathLib|MdePkg/Library/UefiDevicePathLib/UefiDevicePathLib.inf
   FltUsedLib|MdePkg/Library/FltUsedLib/FltUsedLib.inf
+  NULL|MdePkg/Library/CompilerIntrinsicsLib/CompilerIntrinsicsLib.inf
```

Comment:
If this is just for AARCH builds, would it be useful to move the intrinsics to just target those though GCC? Or is this needed for all GCC builds?
```suggestion
[LibraryClasses.common.AARCH64]
  NULL|MdePkg/Library/CompilerIntrinsicsLib/CompilerIntrinsicsLib.inf
```